assets/javascripts/overview.js

```diff
@@ -301,17 +303,33 @@ function addComments(form) {
     controls.style.display = 'inline';
     window.addCommentsModal.hide();
   };
-  fetchWithCSRF(form.action, {method: 'POST', body: new FormData(form)})
+
+  let infoText =
+    'The comments have been created. <a href="javascript: location.reload()">Reload</a> the page to show changes.';
+  let errText = 'The comments could not be added:';
+  if (actionBtn === 'restartAndCommentJobs') {
+    infoText = '<a href="javascript: location.reload()">Reload</a> the page to show restarted jobs.';
+    errText = 'Failed to restart jobs: ';
+  }
+  fetchWithCSRF(reqUrl, {method: 'POST', body: new FormData(form)})
     .then(response => {
       if (!response.ok) throw `Server returned ${response.status}: ${response.statusText}`;
-      addFlash(
-        'info',
-        'The comments have been created. <a href="javascript: location.reload()">Reload</a> the page to show changes.'
-      );
+      addFlash('info', infoText);
       done();
+      return response.json();
+    })
+    .then(resData => {
+      if (resData.errors.length > 0) {
+        //let errFound = resData.errors.map(err => `- ${err.trim()}`).join("\n");
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details